src/passthrough/mod.rs

```diff
@@ -272,8 +282,8 @@ impl HandleData {
     // object is within scope. Otherwise it may cause race window to access wrong target fd.
     // By introducing this method, we could explicitly audit all callers making use of the
     // underlying RawFd.
```

Comment:
The comments above are still referencing RawFd, these should be updated as well?

Also, it'd be better to describe why we're replacing RawFd with BorrowedFd in commit log, what are the benefits.

Reply:
fixed